PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/LinkAzureServiceBeforeRunProvider.java

```diff
@@ -107,8 +107,8 @@ private String getModuleName(@NotNull RunConfiguration runConfiguration) {
             WebAppConfiguration webAppConfiguration = (WebAppConfiguration) runConfiguration;
             final AzureArtifact azureArtifact = AzureArtifactManager.getInstance(runConfiguration.getProject())
                     .getAzureArtifactById(webAppConfiguration.getAzureArtifactType(), webAppConfiguration.getArtifactIdentifier());
-            VirtualFile virtualFile = LocalFileSystem.getInstance().findFileByPath(azureArtifact.getTargetPath());
-            Module module = ProjectFileIndex.getInstance(runConfiguration.getProject()).getModuleForFile(virtualFile, false);
+            final VirtualFile virtualFile = LocalFileSystem.getInstance().findFileByPath(azureArtifact.getTargetPath());
```

Comment:
We shall not get the module from azureArtifact.getTargetPath but not the virtualFile